sig/datadog/appsec/contrib/rack/reactive/request.rbs

```diff
@@ -4,6 +4,8 @@ module Datadog
       module Rack
         module Reactive
           module Request
+            ADDRESSES: ::Array["request.headers" | "request.uri.raw" | "request.query" | "request.cookies" | "request.client_ip"]
```

Comment:
```suggestion
            ADDRESSES: ::Array[::String]
```

Reply:
I have not been putting values in `.rbs` as I believe this would add too much friction in RBS adoption. Also this prevents the possibility to create more precise typing semantics such as `::Array[Reactive::address]`.